### ISSUE TITLE
SA1134 documentation is misleading

### ISSUE THREAD

**Issue by Maxwe11** [DotNetAnalyzers/StyleCopAnalyzers#1924]
The following code triggers SA1134

```c#
public enum DiscriminatedUnionOption
{
    [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "A")] A,

    [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "B")] B
}
```

I know that formatting is off :smile:

**Comment by vweijsters** [comment]
This should trigger SA1134 as far as I can determine, nothing wrong there. The `A` and `B` should be on separate lines.

:question: Is your proposal that the rule should be changed to allow this?

**Comment by sharwell** [comment]
I agree that SA1134 should be reported here. With `A` and `B` at the end of the line, it's unnecessarily hard to read especially on narrow displays.

**Comment by Maxwe11** [comment]
Form [docs](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1134.md)

>Two or more attribute appeared on the same line of code.

I see only one attribute in my code :smile:

**Comment by Maxwe11** [comment]
>The A and B should be on separate lines.

I agree with this but this case not covered by SA1134.

**Comment by vweijsters** [comment]
You have a good point there, the documentation is clearly not correct. SA1134 will report a violation whenever the attribute is on a single line with something else.

**Comment by nbarbettini** [comment]
:question: How's this:

> An attribute is placed on the same line of code as another attribute or element.
> A violation of this rule occurs when two more more attributes are placed on the same line of code, or an attribute is placed on the same line of code as another element.

**Comment by vweijsters** [comment]
@nbarbettini Sounds good to me

**Comment by sharwell** [comment]
Seems clear to me as well

**Comment by nbarbettini** [comment]
:+1: Created a PR - #1933
